CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+# Contributing to hdx-signals
```

Comment:
Would be useful to also have a section on steps for developer setup. This could be something like: 

1. Clone GitHub repository
2. Install R v 4.3.3
3. Install `renv`
4. Install project dependencies using `renv::init()`
5. Make sure you have the necessary environment variables defined (could be in your `.Renviron` file) -- see `ENVIRONMENT.md` for more details